config/routes.rb

```diff
@@ -9,6 +9,7 @@
   post "/login", to: "sessions#create"
   delete "/logout", to: "sessions#destroy"
   get "/dashboard", to: "users#show"
+  post 'update_quantity', to: 'cart_staches#update_quantity'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.